src/91indexeddb.js

```diff
@@ -184,9 +184,26 @@ IDB.createTable = async function (databaseid, tableid, ifnotexists, cb) {
 		throw err;
 	}
 
+	// Get primary key information from the table definition
+	const table = alasql.databases[databaseid].tables[tableid];
+	const pkColumns = table && table.pk && table.pk.columns;
+
+	// Build object store options
+	let storeOptions;
+	if (pkColumns && pkColumns.length === 1) {
+		// Single-column primary key: use keyPath
+		storeOptions = {keyPath: pkColumns[0]};
+	} else if (pkColumns && pkColumns.length > 1) {
+		// Composite primary key: use array keyPath
+		storeOptions = {keyPath: pkColumns};
+	} else {
+		// No primary key: use auto-increment
+		storeOptions = {autoIncrement: true};
```

Comment:
Why are we setting autoincement here? I would expect setting keypath. Maybe im confused.